src/utils/user.authentication.ts

```diff
@@ -46,3 +48,15 @@ export function validateCredentials(credentials: Credentials) {
     );
   }
 }
+
+// secret should be injected
```

Comment:
This is implemented in the next PR https://github.com/strongloop/loopback4-example-shopping/pull/33